src/app/lib/access/kill_switch/kill_switch_repository_mock.dart

```diff
@@ -16,10 +16,9 @@ final class KillSwitchRepositoryMock
 
   bool isKillSwitchActivated = false;
 
-  /// Sets the state of the kill switch.
-  void toggleKillSwitchState() {
-    isKillSwitchActivated = !isKillSwitchActivated;
-
+  /// Sets or toggles the state of the kill switch.
+  void setOrToggleKillSwitchState({bool? isActivated}) {
```

Comment:
This method is a bit weird. It would probably be clearer to have 2 methods: 1 to toggle, 1 to set.